tests/cstor/gtest/test_uzfs.cc

```diff
@@ -3057,6 +3075,11 @@ void mock_tgt_thread(void *arg)
 		uzfs_zvol_set_rebuild_status(zinfo->main_zv,
 		    ZVOL_REBUILDING_INIT);
 
+	if (mgmt_test_case == 25) {
```

Comment:
why this code?

Reply:
To set back the changes made to zinfo volume